gear-test/src/check.rs

```diff
@@ -423,25 +424,27 @@ pub fn check_memory(
 }
 
 fn check_active_programs(
```

Comment:
I would rename the method now.
Something like `check_programs_state` seems good for me.